.gitignore

```diff
@@ -14,5 +14,10 @@ node_modules
 yarn-error.log
 logs
 **/.yarn/cache
+**/.yarn/install-state.gz
 
+# JetBrains IDE
+.idea
 
+# TypeScript
+next-env.d.ts
```

Comment:
I don't think this should be gitignored

Reply:
This does need to be ignored as per —
https://nextjs.org/docs/basic-features/typescript